src/main/flight/pid.c

```diff
@@ -585,6 +635,12 @@ rollAndPitchTrims_t *angleTrim, rxConfig_t *rxConfig)
         }
 
         PTerm -= gyroADCQuant * dynP8[axis] * 0.003f;
+
+        // Pterm low pass
+        if (pidProfile->pterm_cut_hz) {
+            PTerm = filter_pt1(PTerm, &PTerm_state[axis], pidProfile->pterm_cut_hz);
```

Comment:
If pTerm is an OLD variable name, then leave it unchanged for this PR and once this is merged we can rename it.  NEW variables should be in the correct format.